reqwest-tracing/src/reqwest_otel_span_builder.rs

```diff
@@ -61,12 +82,10 @@ pub fn default_on_request_end(span: &Span, outcome: &Result<Response>) {
 #[inline]
 pub fn default_on_request_success(span: &Span, response: &Response) {
     let span_status = get_span_status(response.status());
-    let user_agent = get_header_value("user_agent", response.headers());
```

Comment:
I don't know why the user agent was retrieved from the response. The new key is certainly about the client's user agent, so this is gone.